platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LocalLogoutServlet.java

```diff
@@ -102,4 +112,26 @@ private void deleteJSessionId(HttpServletResponse response) {
     cookie.setComment("EXPIRING COOKIE at " + System.currentTimeMillis());
     response.addCookie(cookie);
   }
+
+  private String getRedirectUriMessage() {
+    String redirectUriMessage = "";
+    if (Strings.isNotBlank(SystemBaseUrl.INTERNAL.getBaseUrl())
+        && Strings.isNotBlank(redirectUri)) {
```

Comment:
❓ Should we instead add the ability to point to any URL which could include an external security system?  I am not sure if we should always assume DDF will handle logout.  For example, you could logout of Keycloak directly instead of redirecting from the DDF logout servlet.

https://www.keycloak.org/docs/latest/securing_apps/index.html#logout

https://www.keycloak.org/docs/latest/securing_apps/index.html#logout